bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_tch_5f_ckm_NLO/st_tch_5f_ckm_NLO_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+import model loop_sm-ckm
```

Comment:
this is not right for 5flavour generation.
You need:
import model loop_sm-ckm_no_b_mass
define p = p b b~
define j = j b b~